spa/src/services/oauth-manager/index.ts

```diff
@@ -53,6 +55,13 @@ async function authenticateInGitHub(onWinClosed: () => void): Promise<void> {
 				}
 			}, 1000);
 		}
+		else{
```

Comment:
This is just one method that uses popup, there are other methods too. So, instead of adding else to just one method here, its better to throw this error in the `popup` method.

Reply:
I think its better to keep it as null 
as if we are throwing the error here and using the popup method into onClick event 
If in case we forget to handle it for some cases 
It will break the UI

Reply:
With this way we would need to add this same else check in all the places where `popup` has been used.
Instead let's just use check inside `popup` method. If the popup is not opened, then set a `popupNotEnabled` value within a constant or localStorage. And simply watch this value in the top level component, maybe `header` or `wrapper`, which will display the error banner when it is set.
Can talk over in details in call...